src/main/resources/jupyter/custom.js

```diff
@@ -0,0 +1,5 @@
+define([
+    'base/js/events'
+], function(events) {
```

Comment:
The entry point to the `custom.js` Jupyter hook. This simply calls `google_sign_in.js`